sql/rpl_gtid_set.cc

```diff
@@ -304,6 +304,22 @@ void Gtid_set::clear_set_and_sid_map() {
   DBUG_VOID_RETURN;
 }
 
+void Gtid_set::remove(const std::vector<rpl_sidno> &sidnos) {
+  if (sid_lock != NULL) sid_lock->assert_some_wrlock();
+  cached_string_length = -1;
+  rpl_sidno max_sidno = get_max_sidno();
+  if (!max_sidno) return;
```

Comment:
avoid implicit conversions
```suggestion
  if (max_sidno == 0) return;
```